backends/cadence/aot/compiler.py

```diff
@@ -16,18 +18,56 @@
     ReplaceScalarTensorWithFullPass,
     ReplaceSqueezeAndUnsqueezeWithViewPass,
 )
+from executorch.backends.cadence.aot.quantizer.fusion_pass import QuantFusion
+from executorch.backends.cadence.aot.quantizer.quantizer import CadenceQuantizer
 from executorch.backends.cadence.aot.utils import model_is_quantized
 from executorch.exir import EdgeCompileConfig, EdgeProgramManager, to_edge
+from torch._export import capture_pre_autograd_graph
 from torch.ao.quantization.pt2e.export_utils import model_is_exported
+from torch.ao.quantization.quantize_pt2e import convert_pt2e, prepare_pt2e
 
 from torch.export import export
 from torch.export.exported_program import ExportedProgram
 
+# pyre-ignore[33]: `InputsArgsType` cannot alias to `Any`.
+InputsArgsType = Tuple[Any, ...]
```

Comment:
Perhaps `object` instead to fix this error message?
```suggestion
InputsArgsType = Tuple[object, ...]
```